components/NotificationPopup/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { useState } from "react";
+
+import Image from "next/image";
+
+import classNames from "classnames/bind";
+
+import { useGetNotifications } from "@/hooks/useGetNotifications";
+
+import styles from "./NotificationPopup.module.scss";
+import BaseNotification from "../Notification/BaseNotification";
+import EmptyNotification from "../Notification/EmptyNotification";
+
+const cn = classNames.bind(styles);
+
+export default function NotificationPopup() {
+  const [isClicked, setIsClicked] = useState(false);
+
+  const { data: allNotifications } = useGetNotifications();
+
+  const handleBellClick = () => {
+    setIsClicked(!isClicked);
+  };
+
+  return (
+    <div className={cn("container")}>
+      <figure onClick={handleBellClick} className={cn("notification-bell")}>
+        <figcaption className={cn("notification-count")}>{allNotifications?.count}</figcaption>
```

Comment:
```suggestion
       { allNotifications?.count > 0 && <figcaption className={cn("notification-count")}>{allNotifications?.count}</figcaption> }
```
P4: `count`가 0인 경우에는 빨간색 숫자를 나타내는 태그가 렌더링되지 않아야 할 것 같아요!